### ISSUE TITLE
Exception for node not being saved in scene

### ISSUE THREAD

**Issue by IceflowRE** [godotengine/godot-docs#7668]
**Your Godot version: 4.1**

**Issue description:**
> If you wish the node or scene to be visible in the scene tree dock and persisted to disk when saving the scene, you need to set the child node's [owner](https://docs.godotengine.org/en/latest/classes/class_node.html#class-node-property-owner) property to the currently edited scene root.

This does **not** work if i set the parent to a node inside an instantiated scene and the owner to the edited scene root.

```
root (owner)
|- instantiatedScene
|  |- innerNode (parent)
|  |  |- myNewNode
```

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/latest/tutorials/plugins/running_code_in_the_editor.html#instancing-scenes

**Comment by betalars** [comment]
I personally think this is a non-issue. Changes to nodes in instanced scenes can only effect the root note of the instance and are not being saved into the main scene. This is very clear to me and does not need an extra explanation. 🤔

**Comment by IceflowRE** [comment]
> I personally think this is a non-issue. Changes to nodes in instanced scenes can only effect the root note of the instance and are not being saved into the main scene. This is very clear to me and does not need an extra explanation. 🤔

Iam not sure if i get you right here. But the text clearly says, if you set the owner to the root, which i did, it is saved to the disk, which it is not. Because of the edge case described above. I attached an example project to be used in Godot 4.4.beta1. Just open scene `foo` and click the button `create child` in the inspector.

[tempgodot.zip](https://github.com/user-attachments/files/18493218/tempgodot.zip)

**Comment by betalars** [comment]
> I am not sure if i get you right here.

tbh I am also not sure if I get you right.

So let me rephrase it in my own words:

1. You have a main scene you are running in.
2. You link an instanced scene within that main scene.
3. You then create a child node in that instanced scene from within the main scene.
4. You want that child note to be saved to the disk?

**Comment by IceflowRE** [comment]
> > I am not sure if i get you right here.
> 
> tbh I am also not sure if I get you right.
> 
> So let me rephrase it in my own words:
> 
>     1. You have a main scene you are running in.
> 
>     2. You link an instanced scene within that main scene.
> 
>     3. You then create a child node in that instanced scene from within the main scene.
> 
>     4. You want that child note to be saved to the disk?

That is exactly what i would expect regarding to the documentation. I would add a short note that this might not be the case when working with instantiated scenes.

**Comment by betalars** [comment]
On the one hand I think this is a very basic concept.

On the other hand this is a section specifically about instancing, so you kind of have a point. Will add it to my thread and think on it.

**Comment by betalars** [comment]
Okay I was reading the [Writing guidelines](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#writing-guidelines) and noticed that section to be lacking in compliance with those, so I ended up rephrasing it after all and therefore resolving this issue.
